### ISSUE TITLE
Add RedwoodJS example

### ISSUE THREAD

**Issue by devkiran** [boxyhq/jackson-examples#21]

**Comment by meenaljain0303** [comment]
I can work on this issue if it's open.

**Comment by devkiran** [comment]
@meenaljain0303 Thanks for showing the interest. Yes, the issue is available. I'll assign this to you. 

Btw have you used RedwoodJS before?

**Comment by meenaljain0303** [comment]
Yes but I am still learning about it, I dont know much about it so if you need me for any other beginner friendly issues as well, i am available.

**Comment by devkiran** [comment]
We have some beginner friendly issues available at https://github.com/boxyhq/saas-starter-kit It is a Next.js based project.
